samples/artifacts/DataSummarization/NamedAggregation/NamedAggregation.siddhi

```diff
@@ -0,0 +1,96 @@
+        d) Select 'CSV file' as the 'Simulation Source'
+        e) Click on 'Add Simulation Source'
+        f) Select 'NamedAggregation' as 'Siddhi App Name'
+        g) Select 'TradeStream' as 'StreamName'
+        h) Click on 'Upload' button under 'CSV file' section and upload the '{SiddhiDistribution_Home}/samples/artifacts/DataSummarization/NamedAggregation/example.csv' file
+        i) Give ',' as the delimiter
+        j) Save the configuration.
+        
+    2) Click on the start button (Arrow symbol) next to the newly created simulator, and wait few seconds to complete the  events sending from the '{SiddhiDistribution_Home}/samples/artifacts/DataSummarization/NamedAggregation/example.csv' file to TradeStream.
+    
+    3)  Click on 'Event Simulator' (double arrows on left tab)
+           a) Click 'Single Simulation' (this will be already selected)
+           b) Select 'NamedAggregation' as Siddhi App Name
+           c) Select 'DashBoardStream' as StreamName
+           d) Provide symbol values
+              - symbol: WSO10
+           e) Send event
+            See the output on the console similar to the following (timestamp will be different):
+           [2019-09-24_13-21-14_228] INFO {io.siddhi.core.stream.output.sink.LogSink} - NamedAggregation : AggregateStockStream : Event{timestamp=1569311474221, data=[WSO10, 1100.0, 220.0], isExpired=false}
+         
+          f) Select 'QueryStream' as StreamName
+          g) Provide triggerId values
+              - triggerId: 1
+          h) Send event
+         See the output on the console similar to the following (timestamp will be different):
+         [2019-09-24_13-21-21_495] INFO {io.siddhi.core.stream.output.sink.LogSink} - NamedAggregation : MaxStockStream : [Event{timestamp=1569311481486, data=[WSO11, 800.0], isExpired=false}, Event{timestamp=1569311481486, data=[WSO16, 700.0], isExpired=false}, Event{timestamp=1569311481486, data=[WSO15, 600.0], isExpired=false}, Event{timestamp=1569311481486, data=[WSO14, 500.0], isExpired=false}, Event{timestamp=1569311481486, data=[WSO10, 500.0], isExpired=false}, Event{timestamp=1569311481486, data=[WSO13, 400.0], isExpired=false}, Event{timestamp=1569311481486, data=[WSO12, 300.0], isExpired=false}]
+         
+         */
+        
+define stream TradeStream(symbol string, price double, volume long,timestamp long);
+
+define stream DashBoardStream(symbol string);
+
+define stream QueryStream(triggerId string);
+
+@sink(type="log")
+define stream AggregateStockStream(symbol string,total double,avgPrice double);
+
+@sink(type="log")
+define stream MaxStockStream(symbol string,maxPrice double);
+
+@Store(type="rdbms",
+       jdbc.url="jdbc:mysql://localhost:3306/production?useSSL=false",
+       username="username",
+       password="password",
+       jdbc.driver.name="com.mysql.jdbc.Driver")
+
+define aggregation TradeAggregation
+from TradeStream
+select symbol, avg(price) as avgPrice, sum(price) as total,max(price) as maxPrice
```

Comment:
```suggestion
select symbol, avg(price) as avgPrice, sum(price) as totalPrice, max(price) as maxPrice
```

Reply:
9ccf6a40df54b295ecf702859004e06cc81436b3 This commit fixed the above correction